roles/openshift_logging/library/openshift_logging_facts.py

```diff
@@ -288,7 +288,7 @@ def facts_for_rolebindings(self, namespace):
             return
         for item in role["subjects"]:
             comp = self.comp(item["name"])
-            if comp is not None and namespace == item["namespace"]:
+            if comp is not None and "namespace" in item and namespace == item["namespace"]:
```

Comment:
you could also use `item.get("namespace")` instead of checking for `"namespace"` in `item` - `get` will return `Null` if the key is not present - unless `Null` is a valid value.

Reply:
Oh, nice. It is shorter. I updated.